torch/csrc/jit/mobile/model_compatibility.cpp

```diff
@@ -203,7 +241,48 @@ ModelCompatibilityInfo ModelCompatibilityInfo::get(
   uint64_t model_bytecode_version =
       _get_model_bytecode_version(bytecode_values);
   auto model_info = _get_model_ops_and_info(bytecode_values);
-  return ModelCompatibilityInfo{model_bytecode_version, model_info};
+  std::vector<IValue> type_table = _get_type_table(bytecode_values);
+  return ModelCompatibilityInfo{model_bytecode_version, model_info, type_table};
+}
+
+std::unordered_set<std::string> splitString(const std::string& str) {
+  std::unordered_set<std::string> output;
```

Comment:
Looks like you are implementing a tokenizer

Reply:
Good point. I added a `contained_type` in type parser instead.